src/Storages/System/StorageSystemZooKeeperConnection.cpp

```diff
@@ -0,0 +1,28 @@
+#include <Interpreters/Context.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <DataTypes/DataTypeDateTime.h>
+#include <Common/ZooKeeper/ZooKeeper.h>
+#include <Storages/System/StorageSystemZooKeeperConnection.h>
+
+namespace DB
+{
+struct ContextSharedPart;
+
+NamesAndTypesList StorageSystemZooKeeperConnection::getNamesAndTypes()
+{
+    return {
+        {"zookeeper_name", std::make_shared<DataTypeString>()},
+        {"zookeeper_address", std::make_shared<DataTypeString>()},
+        {"zookeeper_index", std::make_shared<DataTypeUInt8>()},
+        {"zookeeper_connected_time", std::make_shared<DataTypeDateTime>()},
+    };
```

Comment:
Columns showing `ZooKeeper::expired()`, `ZooKeeper::getApiVersion()`, and `ZooKeeper::getClientID()` will be useful as well